src/SFA.DAS.EmployerIncentives.Web/Extensions/HtmlHelperExtensions.cs

```diff
@@ -21,6 +21,7 @@ public static IHeaderViewModel GetHeaderViewModel(this IHtmlHelper html, bool hi
             var headerModel = new HeaderViewModel(new HeaderConfiguration
                 {
                     EmployerCommitmentsBaseUrl = $"{commitmentsSiteUrl.Scheme}://{commitmentsSiteUrl.Host}/commitments",
+                    EmployerCommitmentsV2BaseUrl = $"{commitmentsSiteUrl.Scheme}://{commitmentsSiteUrl.Host}",
```

Comment:
Am I missing something but this has been added but not used anywhere?

Reply:
It's used in the Nuget package to render the Commitments link
https://github.com/SkillsFundingAgency/das-shared-packages/blob/2e2f957614e17e744235999b07e87855521d6227/SFA.DAS.MA.Shared.UI/SFA.DAS.MA.Shared.UI/Models/HeaderViewModel.cs#L77

Reply:
Ah, outside our domain.